### DISCUSSION TITLE
changing extension of .sql files

### DISCUSSION THREAD

**Discussion by rebred** [sqlpage/SQLPage#730] [Q&A]
how do I change the extension from .sql to  .sqldemo ?

**Comment by lovasoa** [comment]
You can use a reverse proxy with url rewriting for that. 

See [the SQLPage nginx configuration guide](https://sql-page.com/your-first-sql-website/nginx.sql) for reference.
In your case, you may want to write

```nginx
    location / {
        rewrite ^(.*)\.sqldemo$ $1.sql last;
    }
```

But out of curiosity: why are you doing this? What are you building?

**Reply by rebred** [reply]
Thank you we may remove the extension and leave it blank. One of our customer asked to build a shopping cart. Do you think it would be duable in SQLpage ?

**Reply by lovasoa** [reply]
Yes, it sure is doable! 

Check out our authentication example to track user sessions, then create a shopping card table with a reference to the session id.
